sections/exchange/BasicSwap/BasicSwap.tsx

```diff
@@ -10,6 +9,10 @@ import { zIndex } from 'constants/ui';
 import useExchange from 'sections/exchange/hooks/useExchange';
 import { useTranslation } from 'react-i18next';
 
+const ArrowIconComponent = ({ className }: { className: string }) => {
+	return <ArrowIcon className={className} />;
+};
+
```

Comment:
I think I might be missing something (maybe a TS issue), but is there a reason we aren't just doing `<ArrowIcon className="arrow" />` directly?